src/interface_py/h2o4gpu/solvers/truncated_svd.py

```diff
@@ -124,6 +124,7 @@ def fit_transform(self, X, y=None):
         param.random_state = self.random_state
         param.verbose = self.verbose
         param.gpu_id = self.gpu_id
+        param.whiten = False #Whitening is not exposed for tsvd yet
```

Comment:
I might/might not expose this parameter. Just want to leave here as a reminder for myself.